[quant][docs] Add types for scale and zero_point tensor for torch.fake_quantize_per_channel_affine docs

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #85733

Summary:
Fixes: https://github.com/pytorch/pytorch/issues/85525

Test Plan:
visual inspection for the docs page

Reviewers:

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
[quant][docs] Add types for scale and zero_point tensor for torch.fake_quantize_per_channel_affine docs

Summary:
Fixes: https://github.com/pytorch/pytorch/issues/85525

Test Plan:
visual inspection for the docs page

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

z-a-f reviewed (approved):
Does sphinx support that way of defining the arg types? link: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

jerryzh168 committed:
Update on "[quant][docs] Add types for scale and zero_point tensor for torch.fake_quantize_per_channel_affine docs"


Summary:
Fixes: https://github.com/pytorch/pytorch/issues/85525

Test Plan:
visual inspection for the docs page

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
> Does sphinx support that way of defining the arg types? link: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

probably not, moved the dtypes to descriptions

jerryzh168 commented:
@pytorchbot merge -g
